src/function/scalar/sequence/nextval.cpp

```diff
@@ -121,8 +121,7 @@ static unique_ptr<FunctionData> NextValBind(ClientContext &context, ScalarFuncti
 		// evaluate the constant and perform the catalog lookup already
 		auto seqname = ExpressionExecutor::EvaluateScalar(*arguments[0]);
 		if (!seqname.IsNull()) {
-			D_ASSERT(seqname.type().id() == LogicalTypeId::VARCHAR);
-			auto qname = QualifiedName::Parse(StringValue::Get(seqname));
+			auto qname = QualifiedName::Parse(StringValue::Get(seqname.ToString()));
```

Comment:
The `StringValue::Get` is no longer required (and is actually causing a number of extra operations here). Could you remove this as well?

Reply:
yes, my apologies